floss/identify.py

```diff
@@ -173,11 +173,19 @@ def find_decoding_function_features(vw, functions, disable_progress=False) -> Tu
                 function_data["features"].append(feature)
 
             function_data["score"] = get_function_score_weighted(function_data["features"])
+            decoding_candidate_functions[function_address] = function_data
 
-            logger.debug("analyzed function 0x%x - total score: %.3f", function_address, function_data["score"])
-            for feat in function_data["features"]:
-                logger.trace("  %s", feat)
+    for fva, function_data in decoding_candidate_functions.items():
+        logger.debug("analyzed function 0x%x - total score: %.3f", fva, function_data["score"])
+    
+    for feat in function_data["features"]:
+        logger.trace("  %s", feat)
 
-            decoding_candidate_functions[function_address] = function_data
+    decoding_function_summaries = [
+    f"0x{fva:x} ({data['score']:.3f}, xrefs_to: {data['xrefs_to']})"
+    for fva, data in decoding_candidate_functions.items()
+    ]
+    summary_string = textwrap.fill(", ".join(decoding_function_summaries))
+    logger.info(f"Identified decoding functions:\n{summary_string}")
```

Comment:
Do you feel strongly that we need this here and that it adds value?

Reply:
We can comment it for now , and use it for checking at a later point .